main.js

```diff
@@ -1,97 +1,182 @@
-}
-function getIconURL(iconName) {
-  return ` http://openweathermap.org/img/wn/${iconName}@2x.png`;
-}
+    let mainTime = document.createElement("div");
+    mainTime.className = "main-time";
+    mainTime.innerHTML = time;
 
-function converTemp(unit) {
-  for (let i = 0; i < myList.length; i++) {
-    const element = myList[i];
-    element.unit = unit;
-    if (unit === "°F") {
-        element.temp = ((element.temp * 9) / 5 + 32).toFixed();
-    }
-    else{
-        element.temp = ((element.temp - 32) * 5/9).toFixed();
-    }
+    let mainImg = document.createElement("img");
+    mainImg.src = iconSrc;
+
+    let mainVariables = document.createElement("div");
+    mainVariables.className = "main-variables";
+    let layoutTemperature = document.createElement("div");
+    layoutTemperature.className = "layout-temperature";
+    let layoutHumidity = document.createElement("div");
+
+    let mainTemperature = document.createElement("div");
+    mainTemperature.innerHTML = temp;
+    mainTemperature.className = "main-temperature";
+
+    let mainUnit = document.createElement("div");
+    mainUnit.innerHTML = unit;
+    mainUnit.className = "main-unit";
+
+    layoutTemperature.appendChild(mainTemperature);
+    layoutTemperature.appendChild(mainUnit);
+
+    let mainHumidity = document.createElement("div");
+    mainHumidity.innerHTML = `${humi}%`;
+    mainHumidity.className = "main-humidity";
+    layoutHumidity.appendChild(mainHumidity);
+    mainVariables.appendChild(layoutTemperature);
+    mainVariables.appendChild(layoutHumidity);
+
+    let singleFullReport = document.createElement("div");
+    singleFullReport.className = "single-full-report";
+    let singleReportInfo = document.createElement("div");
+    singleReportInfo.className = "single-report";
+    singleReportInfo.appendChild(mainImg);
+    singleReportInfo.appendChild(mainVariables);
+    singleFullReport.appendChild(mainTime);
+    singleFullReport.appendChild(singleReportInfo);
+    forecastList.appendChild(singleFullReport);
   }
-}
-function handleClick(myRadio) {
+};
+
+const handleClick = (myRadio) => {
+  myRadio.checked = true;
   if (myRadio.value !== myList[0].unit) {
-    converTemp(myRadio.value);
+    console.log(myRadio.value);
+    myRadio.value === "°C"
+      ? (radioBfahrenheit.checked = false)
+      : (radioBCelsius.checked = false);
+    converTemperature(myRadio.value);
   }
   drawList();
-}
+};
 
+const getInfoAPI = (cityName, keyAPI) => {
+  let promise = fetch(
+    `http://api.openweathermap.org/data/2.5/forecast?q=${cityName}&appid=${keyAPI}`
+  );
+  promise
+    .then(function (response) {
+      if (response.status !== 200) {
+        console.log(
+          "Looks like there was a problem. Status Code: " + response.status
+        );
+        return;
+      }
+      //   // Examine the text in the response
+      response.json().then(function (data) {
+        radioBCelsius.checked = true;
+        radioBfahrenheit.checked = false;

```

Comment:
Tienes que ser consistente con lsa mayusculas de tus variables:
```suggestion
        radioBCelsius.checked = true;
        radioBFahrenheit.checked = false;
```